src/js/angular/clustermanagement/controllers.js

```diff
@@ -241,7 +241,7 @@ function ClusterManagementCtrl($scope, $http, $q, toastr, $repositories, $uibMod
                 })
                 .catch((error) => {
                     const failMessage = $translate.instant('cluster_management.delete_cluster_dialog.notifications.fail_delete');
-                    const failedNodesList = Object.keys(error.data)
+                    const failedNodesList = error.status === 403 ? getError(error.data, error.status) : Object.keys(error.data)
```

Comment:
Is it only 403 that needs to be handled like this?

IMO all errors should be handled with `getError()`, except the one that returns the failed delete from the cluster.